tool/jt.rb

```diff
@@ -709,42 +709,84 @@ def compile_cext(name, ext_dir, target, *clang_opts)
+        sh 'git', 'push', '--no-verify', bb, ":#{remote_branch}"
+      end
+      sh 'git', 'fetch', bb, '--prune'
     end
 
-    upstream = remote_urls.find { |r, u| u.include? 'graalvm/truffleruby' }.first
-    bb       = remote_urls.find { |r, u| u.include? 'ol-bitbucket' }.first
+    def pr_push(*args)
+      remote_urls = self.remote_urls
+      upstream    = upstream(remote_urls)
+      bb          = bb(remote_urls)
 
-    # Fetch PRs on GitHub
-    fetch = "+refs/pull/*/head:refs/remotes/#{upstream}/pr/*"
-    out, _err = sh 'git', 'config', '--get-all', "remote.#{upstream}.fetch", capture: true
-    sh 'git', 'config', '--add', "remote.#{upstream}.fetch", fetch unless out.include? fetch
-    sh 'git', 'fetch', upstream
+      # Fetch PRs on GitHub
+      fetch     = "+refs/pull/*/head:refs/remotes/#{upstream}/pr/*"
+      out, _err = sh 'git', 'config', '--get-all', "remote.#{upstream}.fetch", capture: true
+      sh 'git', 'config', '--add', "remote.#{upstream}.fetch", fetch unless out.include? fetch
+      sh 'git', 'fetch', upstream
 
-    pr_number = args.first
-    if pr_number
-      github_pr_branch = "#{upstream}/pr/#{pr_number}"
-    else
-      github_pr_branch = begin
-        out, _err = sh 'git', 'branch', '-r', '--contains', 'HEAD', capture: true
-        candidate = out.lines.find { |l| l.strip.start_with? "#{upstream}/pr/" }
-        candidate && candidate.strip.chomp
-      end
+      pr_number = args.first
+      if pr_number
+        github_pr_branch = "#{upstream}/pr/#{pr_number}"
+      else
+        github_pr_branch = begin
+          out, _err = sh 'git', 'branch', '-r', '--contains', 'HEAD', capture: true
+          candidate = out.lines.find { |l| l.strip.start_with? "#{upstream}/pr/" }
+          candidate && candidate.strip.chomp
+        end
 
-      unless github_pr_branch
-        puts 'Could not find HEAD in any of the GitHub pull-requests.'
-        exit 1
+        unless github_pr_branch
+          puts 'Could not find HEAD in any of the GitHub pull-requests.'
+          exit 1
+        end
+
+        pr_number = github_pr_branch.split('/').last
       end
 
-      pr_number = github_pr_branch.split('/').last
+      sh 'git', 'push', '--force', bb, "#{github_pr_branch}:refs/heads/github/pr/#{pr_number}"
+    end
+
+    def bb(remote_urls)
+      remote_urls.find { |r, u| u.include? 'ol-bitbucket' }.first
     end
 
-    sh 'git', 'push', '--force', bb, "#{github_pr_branch}:refs/heads/github/pr/#{pr_number}"
+    def upstream(remote_urls)
+      remote_urls.find { |r, u| u.include? 'graalvm/truffleruby' }.first
+    end
+
+    def remote_urls
+      out, _err   = sh 'git', 'remote', capture: true
+      remotes     = out.split
+      remotes.map do |remote|
+        out, _err = sh 'git', 'config', '--get', "remote.#{remote}.url", capture: true
+        [remote, out.chomp!]
+      end
+    end
+
+    extend self
+  end
+
+  def pr(*args)
+    command = args.first
+    if command == 'clear'
```

Comment:
Should it be called `clean` or handled as an alias?
`clear` sounds like it would just delete everything to me.